createContacts/index.ts

```diff
@@ -0,0 +1,48 @@
+import {AzureFunction, Context, HttpRequest} from '@azure/functions';
+import {contactRecord} from '../models/contact-Record';
+import {newContact} from '../models/new-Contact';
+import {getUserId, GuidClass, phoneValidator} from '../Common/Utils';
+
+const httpTrigger: AzureFunction = async function (
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  let idValue = new GuidClass();
+  let idTemp = JSON.stringify(idValue.id);

```

Comment:
is idValue.id not already a string?

Reply:
Normally when creating a contact, they would be associated with an existing user. Creating a contact shouldn't create a user. It would make more sense to have the userId as a param for this endpoint.